org.kde.kronometer.json

```diff
@@ -0,0 +1,38 @@
+{
+    "id": "org.kde.kronometer",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kronometer",
+    "rename-icon": "kronometer",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
```

Comment:
Seems to want to access some dbus to disable the screenlocker, can you investigate?

Could not inhibit screen locker: QDBusError("org.freedesktop.DBus.Error.ServiceUnknown", "org.freedesktop.DBus.Error.ServiceUnknown")